src/main/java/trackitnus/model/lesson/DayOfWeek.java

```diff
@@ -13,8 +12,8 @@
     Sat;
     private static final int SIZE = 7;
 
-    public static LessonWeekday getLessonWeekDay(LocalDate date) {
-        DayOfWeek dayOfWeek = date.getDayOfWeek();
+    public static DayOfWeek getLessonWeekDay(LocalDate date) {
+        java.time.DayOfWeek dayOfWeek = date.getDayOfWeek();
```

Comment:
Wouldn't it be better to keep the import so you dont need to type java.time.DayOfWeek